app/templates/content.hbs

```diff
@@ -83,12 +83,8 @@
                     </div>
                     <div class="p-t-xs">
                         <h4 class="p-v-md f-w-md"><strong>{{t "global.abstract"}}</strong></h4>
-                        {{#if useShortenedDescription}}
-                            <p class="abstract">{{description}}</p>
-                            <button class='btn-link' {{action 'expandAbstract'}}>{{if expandedAbstract 'See less' 'See more'}}</button>
-                        {{else}}
-                            <p class="abstract">{{node.description}}</p>
-                        {{/if}}
+                        <p class="abstract {{unless expandedAbstract 'abstract-truncated'}}">{{if expandedAbstract node.description description}}</p>
+                        <button class='btn-link' hidden={{not hasShortenedDescription}} {{action 'expandAbstract'}}>{{t (if expandedAbstract 'content.see_less' 'content.see_more')}}</button>
```

Comment:
Something we may want to watch for: will this button text become part of the indexed abstracts for our preprints?